tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -4881,19 +4881,18 @@ public function propfindResultShouldContainEntries(
 				continue;
 			}
 			if ($method === "REPORT") {
-				$fileFound = $this->findEntryFromSearchResponse(
+				$xmlItem = $this->findEntryFromSearchResponse(
 					$resource
 				);
+				$fileFound = !\is_bool($xmlItem) ? $xmlItem->xpath("d:propstat//oc:name") : $xmlItem;
```

Comment:
```suggestion
				$fileFound = $this->findEntryFromSearchResponse(
					$resource
				);
```

Reply:
I think the function return mixed type where string|bool|object. May be we check all of those as per required.

Reply:
so does this function `$fileFound = $this->findEntryFromPropfindResponse(` just below. Should be fine I think